Makefile

```
@@ -8,7 +10,7 @@ run: ## Run the game
 	@swipl --quiet -t "main($(level), \"$(file)\", $(players), $(fac))" -s game.pl 
     
 test: ## Run tests
-	@swipl --quiet -t "load_test_files([]), run_tests." -s game.pl 
+	@swipl -t "load_test_files([]), run_tests." -s game.pl 
  
 coverage: ## Run tests and display coverage
 	@swipl --quiet -t "use_module(library(test_cover)), load_test_files([]), show_coverage(run_tests)." -s utils.pl -s game.pl
```

Change to top level by goals in order to hide the swipl welcome message. And remove duplicated -s
```suggestion
	@swipl -g "use_module(library(test_cover)), load_test_files([]), show_coverage(run_tests)." -g halt -s game.pl
```